src/main/java/com/swervedrivespecialties/exampleswerve/subsystems/DrivetrainSubsystem.java

```diff
@@ -132,8 +146,59 @@ public void resetGyroscope() {
         gyroscope.setAdjustmentAngle(gyroscope.getUnadjustedAngle());
     }
 
-    @Override
```

Comment:
I believe you didn't mean to get rid of this code. Without this code, the Chassis won't drive. Return the InitDefaultCommand

Reply:
I don't think subsystembase needs a default command?

Reply:
initDefaultCommand has been removed; subsystems no longer need to “know about” their default commands, which are instead registered directly with the CommandScheduler. The new setDefaultCommand method simply wraps the CommandScheduler call.

Subsystems no longer “know about” the commands currently requiring them; this is handled exclusively by the CommandScheduler. A convenience wrapper on the CommandScheduler method is provided, however.

^From the WPILIB site